onnxruntime/core/session/ort_apis.h

```diff
@@ -354,7 +354,7 @@ ORT_API_STATUS_IMPL(CreateOpAttr,
 ORT_API(void, ReleaseOpAttr, _Frees_ptr_opt_ OrtOpAttr* op_attr);
 
 ORT_API_STATUS_IMPL(CreateOp,
-                    _In_ const OrtKernelInfo* info,
+                    _In_ const OrtExecutionProvider* ep,
```

Comment:
>rtExecutionProvider* ep, [](http://example.com/codeflow?start=32&length=24)

This is chaning the existing entry point. Should we not change the struct version?

Reply:
The feature will be made public officially in the coming release of 1.12.